lib/bolt/executor.rb

```diff
@@ -170,6 +170,7 @@ def run_task(targets, task, arguments, options = {}, &callback)
       log_action(description, targets) do
         notify = proc { |event| @notifier.notify(callback, event) if callback }
         options = { '_run_as' => run_as }.merge(options) if run_as
+        options = options.merge('_task' => task.name)
```

Comment:
There isn't a good reason to merge and create a new hash here. We can just options['_task'] = task.name